src/client/sandbox/node/window.js

```diff
@@ -222,6 +222,16 @@ export default class WindowSandbox extends SandboxBase {
         };
         window.Image.prototype = nativeMethods.Image.prototype;
 
+        window.Function = function (...args) {
+            var lastArgIndex = args.length - 1;
+
+            if (typeof args[lastArgIndex] === 'string')
+                args[lastArgIndex] = processScript(args[lastArgIndex], false, false);
+
+            return nativeMethods.Function.apply(this, args);
+        };
+        window.Function.prototype = nativeMethods.Function.prototype;
+
```

Comment:
Add test for this line. See https://github.com/DevExpress/testcafe-hammerhead/commit/589d08b0aed68e108168b2d4bae5df10fdea271c